File: Web/src/serverApi/models/excelModel/HiExcelExportTemplateEntity.ts
import { StringHelper } from '@/helper/stringHelper';

export class HiExcelExportTemplateEntity {
  /**
   * 导出任务标题
   */
  Title = ``;

  /**
   * TemplateId[必填参数]
   */
  TemplateId = StringHelper.genGuid();

  /**
   * FileUrl
   */
  FileUrl = '';

  /**
   * ModiName
   */
  ModiName = '';

  /**
   * SortNum
   */
  SortNum = 0;

  /**
   * ModiTime
   */
  ModiTime: Date | string = new Date();
}

File: Server/HiSql.GUI/Repository/Models/HiExcelExportTemplateEntity.cs
using System;
namespace HiSql.GUI.Repository.Models
{
    public class HiExcelExportTemplateEntity
    {
        /// <summary>
        /// 导出任务标题
        /// </summary>
        public string Title { get; set; }

        /// <summary>
        /// TemplateId[必填参数]
        /// <summary>
        public string TemplateId { get; set; }

        /// <summary>
        /// FileUrl
        /// <summary>
        public string FileUrl { get; set; }

        /// <summary>
        /// ModiName
        /// <summary>
        public string ModiName { get; set; }

        /// <summary>
        /// SortNum
        /// <summary>
        public int SortNum { get; set; }

        /// <summary>
        /// ModiTime
        /// <summary>
        public DateTime ModiTime { get; set; }

    }
}

